beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -1399,7 +1459,9 @@ mod tests {
         let mut dst = BytesMut::with_capacity(1024);
 
         // Insert length-prefix
-        uvi_codec.encode(MAX_RPC_SIZE + 1, &mut dst).unwrap();
+        uvi_codec
+            .encode(Spec::default_spec().max_chunk_size as usize + 1, &mut dst)
```

Comment:
you are creating a `Spec::default_spec()` here and another one in line 1475. Just create one `let spec = Spec::default_spec()` above and use it here and below too